src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
+      return null;
+    }
+
+    invertedRobot[robot[key]] = key;
```

Comment:
When assigning the key to the `invertedRobot`, it's a good practice to ensure that the value is a string since object keys in JavaScript are always strings or symbols. If the key is not a string, it will be coerced to a string, which could lead to unexpected behavior if the key is, for example, a number. You can use `String(key)` to explicitly convert it to a string.